pkg/operator/controllers/genevalogging/const.go

```diff
@@ -73,7 +73,7 @@ const (
 [FILTER]
 	Name grep
 	Match containers
-	Regex NAMESPACE ^(?:default|kube-.*|openshift|openshift-.*)$
+	Regex NAMESPACE ^(?:default|kube-.*|openshift|^(?!openshift-(logging|gitops))(openshift-.*))$
```

Comment:
There is no need for the second ^ because the first one is outside the non-capturing group.